multiversx_sdk_cli/cli_delegation.py

```diff
@@ -369,66 +390,233 @@ def ensure_arguments_are_provided_and_prepared(args: Any):
     cli_shared.prepare_nonce_in_args(args)
 
 
```

Comment:
can you set the nonce on the sender and then use it from there?

Reply:
Yes, done